modules/RTC/JitsiLocalTrack.js

```diff
@@ -174,6 +191,11 @@ export default class JitsiLocalTrack extends JitsiTrack {
         RTCUtils.addListener(RTCEvents.DEVICE_LIST_WILL_CHANGE, this._onDeviceListWillChange);
 
         this._initNoDataFromSourceHandlers();
+
+        if (this._videoBlurEnabled === true && this._jitsiStreamBlurEffect) {
```

Comment:
I'm asking out of curiosity - Why don't you include this logic into the `if` that initialises  `this. _jitsiStreamBlurEffect`?

Reply:
Moved code to initialization if branch